client/src/listings/Listings.js

```diff
@@ -123,27 +124,32 @@ function Listings(props) {
   return (
     <div className='listings-container'>
       <Typography component='h1' variant='h5'>
-        Find Services:
+        Find Classifieds & Home Share Links:
       </Typography>
       <form className={classes.form} noValidate>
         <Grid container spacing={3} direction='row' justify='space-evenly' alignItems='flex-end'>
           <Grid item xs>
             <InputLabel>Select a Category:</InputLabel>
-            <Select
-              className='listing-select'
-              name='categoryId'
-              value={categoryId}
-              required
-              onChange={handleCategoryChange}
-            >
-              {categoryOptions.map((option) => (
-                <MenuItem key={option.id} value={option.id}>
-                  {option.name}
-                </MenuItem>
-              ))}
-            </Select>
+            <Select className="listing-select" name='categoryId' native defaultValue='' id='categoryId'>
+                    <option value='' />
+                    {categoryOptions.length > 0 && Object.keys(subcategoryOptions).length > 0 ? (
+                      categoryOptions.map((cat) => (
+                        <optgroup key={cat.id} label={`${cat.name}`}>
+                          {subcategoryOptions[cat.id] ? (
+                            subcategoryOptions[cat.id].map((subcat) => (
+                              <option key={subcat.id} value={subcat.id}>{`${subcat.name}`}</option>
+                            ))
+                          ) : (
+                            <></>
+                          )}
+                        </optgroup>
+                      ))
+                    ) : (
+                      <option value='' />
+                    )}
+                  </Select>
```

Comment:
This dropdown looks nice but doesn't actually do anything. Did you forget to hook up the functionality for updating the search results?

Reply:
I thought I only need to do the static one, I will update it